### ISSUE TITLE
[CLOSED] Vanish in Flare

### ISSUE THREAD

**Issue by netherspite** [netherspite/cc-buglist#17]
<a href="https://github.com/ccshiro"><img src="https://avatars2.githubusercontent.com/u/8135895?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [ccshiro](https://github.com/ccshiro)**    
_Tuesday Nov 04, 2014 at 15:57 GMT_  
_Originally opened as https://github.com/ccshiro/cc-buglist/issues/17_

----

Question: Should a rogue be able to use vanish in a hunter's flare? In MoP you couldn't, but reports suggests in TBC you could. A video of this happening would be sufficient evidence.

**Comment by netherspite** [comment]
<a href="https://github.com/Pyrossword"><img src="https://avatars1.githubusercontent.com/u/9559330?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Pyrossword](https://github.com/Pyrossword)**  
_Tuesday Nov 04, 2014 at 21:04 GMT_

----

I'll need someone else to confirm this to me, but I checked all patch changes in TBC and have not seen mention of Rogues vanishing in Hunters flare.

**Comment by netherspite** [comment]
<a href="https://github.com/ccshiro"><img src="https://avatars2.githubusercontent.com/u/8135895?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ccshiro](https://github.com/ccshiro)**  
_Tuesday Nov 04, 2014 at 21:10 GMT_

----

Penol linked this video to me: http://www.youtube.com/watch?v=srwTw3BoU3s#t=27

It's 3.3, however, and while I think it's a fairly strong indication, if someone is able to find a TBC video that would settle the issue once and for all.

**Comment by netherspite** [comment]
<a href="https://github.com/Pyrossword"><img src="https://avatars1.githubusercontent.com/u/9559330?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Pyrossword](https://github.com/Pyrossword)**  
_Tuesday Nov 04, 2014 at 21:13 GMT_

----

I will try to surf YT for the video and hopefully will find it.

**Comment by netherspite** [comment]
<a href="https://github.com/Pyrossword"><img src="https://avatars1.githubusercontent.com/u/9559330?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Pyrossword](https://github.com/Pyrossword)**  
_Tuesday Nov 04, 2014 at 21:18 GMT_

----

I don't know if you read the description of the video Shiro but this caught my eye.

'Edit: As of 10/25/2009 this change has been reverted and no longer works'

**Comment by netherspite** [comment]
<a href="https://github.com/MakDizdar"><img src="https://avatars1.githubusercontent.com/u/9559717?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [MakDizdar](https://github.com/MakDizdar)**  
_Tuesday Nov 04, 2014 at 21:59 GMT_

----

No, you could not.
You are stealthed for 0.5 seconds, that is not enough for a cheap shot considering that the global cooldown is 0.5 seconds. The amount of time you have to cheapshot collides with the global cooldown.

To sum up: No, i have no evidence of this and my memory could fail but I am quite sure that you could not vanish in a flare. You could still cast vanish but you got immediately revealed, maybe just enough time to escape a flare.

**Comment by netherspite** [comment]
<a href="https://github.com/Skibbli"><img src="https://avatars2.githubusercontent.com/u/9157636?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Skibbli](https://github.com/Skibbli)**  
_Tuesday Nov 04, 2014 at 22:06 GMT_

----

http://www.youtube.com/watch?v=EbwetgVudCM - at 6:35 you see Neilyo use vanish with the flare debuff on him (middle left of the screen) and immune pops up. He stays visible but the slow is removed.

**Comment by netherspite** [comment]
<a href="https://github.com/huggarn"><img src="https://avatars3.githubusercontent.com/u/6495985?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [huggarn](https://github.com/huggarn)**  
_Tuesday Nov 04, 2014 at 22:18 GMT_

----

MakDizar Vanish caused GCD? On AT they could insta vanish/sap you.

**Comment by netherspite** [comment]
<a href="https://github.com/Lobogo"><img src="https://avatars0.githubusercontent.com/u/9566610?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Lobogo](https://github.com/Lobogo)**  
_Wednesday Nov 05, 2014 at 07:47 GMT_

----

Yes indeed you COULD use vanish in a flare, but all it did was remove the slowing effects (if any) and you would be insta revealed again
And as said above, due to global CD you wouldnt be able to use any opening moves

**Comment by netherspite** [comment]
<a href="https://github.com/Barroth"><img src="https://avatars2.githubusercontent.com/u/9391338?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Barroth](https://github.com/Barroth)**  
_Wednesday Nov 05, 2014 at 08:06 GMT_

----

nice find Skibbli so he gets a immune message when stealth/vanish buffs should get applied, drops combat and only removes snare

**Comment by netherspite** [comment]
<a href="https://github.com/pnoia"><img src="https://avatars1.githubusercontent.com/u/9556779?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [pnoia](https://github.com/pnoia)**  
_Wednesday Nov 05, 2014 at 10:38 GMT_

----

Vanish doesnt trigger GCD so the reasoning behind not being able to use any opening moves is false. The explanation to this would be flare revealing the rogue instantly thereby removing the stealth effect before being unable to perform any action.

Here is a video of vanish not triggering the GCD. You can see the rogue uses garrote instantly after using vanish.

5:08 http://youtu.be/2rmCkvpyzEw?t=5m8s

**Comment by netherspite** [comment]
<a href="https://github.com/Lobogo"><img src="https://avatars0.githubusercontent.com/u/9566610?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Lobogo](https://github.com/Lobogo)**  
_Wednesday Nov 05, 2014 at 11:04 GMT_

----

in that case, yes, its the insta reveal that prevents you from using openers while under flare effect

**Comment by netherspite** [comment]
<a href="https://github.com/ccshiro"><img src="https://avatars2.githubusercontent.com/u/8135895?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ccshiro](https://github.com/ccshiro)**  
_Wednesday Nov 05, 2014 at 14:00 GMT_

----

Guess that settles it. Thanks.
